### DISCUSSION TITLE
cross-hair cursor extensions

### DISCUSSION THREAD

**Discussion by zumbruch** [ControlSystemStudio/phoebus#2858] [Ideas]
Would it be possible to extend and/or configure the cross-hair cursor markers?
E.g. PV name, trace name, display name, ... 

Up to my knowledge only value and severity currently included

![image](https://github.com/ControlSystemStudio/phoebus/assets/1180252/346f790c-e08b-4ddb-9599-60f432866f58)

**Comment by kasemir** [comment]
If you want to implement that, it would certainly need to be configurable, i.e., not change the content for everybody, because what's in there can already be too much and use too much space within the plot when you have multiple traces.

There are already annotations which have configurable content, which can be useful if you want to highlight some specific point/value/edge/.. for a screenshot
